sktime/forecasting/nnetsaucemts.py

```diff
@@ -0,0 +1,738 @@
+        #
+        # specify one or multiple authors and maintainers, only for sktime contribution
+        "authors": ["thierrymoudiki"],  # authors, GitHub handles
+        "maintainers": ["thierrymoudiki"],  # maintainers, GitHub handles
+        # author = significant contribution to code at some point
+        #     if interfacing a 3rd party estimator, ensure to give credit to the
+        #     authors of the interfaced estimator
+        # maintainer = algorithm maintainer role, "owner" of the sktime class
+        #     for 3rd party interfaces, the scope is the sktime class only
+        # remove maintainer tag if maintained by sktime core team
+        #
+        # do not change these:
+        # (look at advanced templates if you think these should change)
+        "y_inner_mtype": "pd.DataFrame",
+        "X_inner_mtype": "pd.DataFrame",
+        "ignores-exogeneous-X": False,
+        "capability:pred_int": True,
+        "capability:pred_int:insample": True,
+        "requires-fh-in-fit": False,
+        "python_dependencies": [
+            "nnetsauce",
+            "joblib",
+            "matplotlib",
+            "numpy",
+            "pandas",
+            "requests",
+            "scipy",
+            "scikit-learn",
+            "statsmodels",
+            "threadpoolctl",
+            "tqdm",
+        ],
+    }
+
+    # todo: add any hyper-parameters and components to constructor
+    def __init__(
+        self,
+        regressor=RidgeCV(),
+        n_hidden_features=5,
+        activation_name="relu",
+        a=0.01,
+        nodes_sim="sobol",
+        bias=True,
+        dropout=0,
+        direct_link=True,
+        n_clusters=2,
+        cluster_encode=True,
+        type_clust="kmeans",
+        type_scaling=("std", "std", "std"),
+        lags=1,
+        type_pi="kde",
+        block_size=None,
+        replications=None,
+        kernel="gaussian",
+        agg="mean",
+        seed=123,
+        backend="cpu",
+        verbose=0,
+        show_progress=True,
+    ):
+        self.obj = self.regressor = regressor
+        self.n_hidden_features = n_hidden_features
+        self.activation_name = activation_name
+        self.a = a
+        self.nodes_sim = nodes_sim
+        self.bias = bias
+        self.dropout = dropout
+        self.direct_link = direct_link
+        self.n_clusters = n_clusters
+        self.cluster_encode = cluster_encode
+        self.type_clust = type_clust
+        self.type_scaling = type_scaling
+        self.lags = lags
+        self.type_pi = type_pi
+        self.block_size = block_size
+        self.replications = replications
+        self.kernel = kernel
+        self.agg = agg
+        self.seed = seed
+        self.backend = backend
+        self.verbose = verbose
+        self.show_progress = show_progress
+
+        # leave this as is
+        super().__init__()
+
+        from nnetsauce import MTS
+
+        self.fitter = MTS(
```

Comment:
Will it make sense to indicate `fitter` as internal, e.g. using `self._fitter`?

Also, I think in most estimators we do it inside `_fit` instead of `__init__`, but not sure if that's a rule or convention. @fkiraly can confirm.